### DISCUSSION TITLE
Soundshed tones saved to amp channels as default patches

### DISCUSSION THREAD

**Discussion by SoloArchitect** [soundshed/soundshed-app#46] [Q&A]
I have just started using the app. Very nice! Once I have selected a tone and tweaked it to my liking, is there a way to save it to a channel on the spark? 

I understand there is lots of work to do to get this integrated with tonecloud, what would be the easiest way to get tones tweaked or used in soundshed onto the amp or my spark account?

**Comment by webprofusion-chrisc** [comment]
Currently the easiest way is just to long press one of the (physical) amp channel buttons, which will store the current preset. I'll eventually add the option to save directly to the amp, I just haven't gotten round to it yet.
